services/core-web/src/components/common/wrappers/AddPartyComponentWrapper.js

```diff
@@ -99,7 +99,7 @@ export class AddPartyComponentWrapper extends Component {
           <Radio.Group
             defaultValue
             size="large"
-            value={this.props.isPerson}
+            value={this.state.isPerson}
             onChange={this.togglePartyChange}
```

Comment:
During the merge conflict, this came up as a linting error, "this.props.isPerson"